frontend/src/citizen-frontend/calendar/MonthElem.tsx

```diff
@@ -71,28 +74,33 @@ export default React.memo(function MonthElem({
 }: MonthProps) {
   const i18n = useTranslation()
 
-  const [monthlySummaryInfoOpen, setMonthlySummaryInfoOpen] = useState(false)
-  const onMonthlySummaryInfoClick = useCallback(
-    () => setMonthlySummaryInfoOpen((prev) => !prev),
-    []
-  )
   const displaySummary = featureFlags.timeUsageInfo && childSummaries.length > 0
+  const displayAlert =
+    displaySummary &&
+    childSummaries.some(
+      ({ reservedMinutes, usedServiceMinutes, serviceNeedMinutes }) =>
+        reservedMinutes > serviceNeedMinutes ||
+        usedServiceMinutes > serviceNeedMinutes
+    )
+  const { summaryInfoOpen, toggleSummaryInfo } = useSummaryInfo(childSummaries)
```

Comment:
`displayAlert` could also be returned by the hook. That way, the `usedServiceMinutes > serviceNeedMinutes` logic would be only in one place.